c7n/resources/apigw.py

```diff
@@ -122,14 +124,26 @@ class RestApi(query.QueryResourceManager):
 
     class resource_type(query.TypeInfo):
         service = 'apigateway'
-        arn_type = 'restapis'
+        arn_type = '/restapis'
         enum_spec = ('get_rest_apis', 'items', None)
         id = 'id'
         name = 'name'
         date = 'createdDate'
         dimension = 'GatewayName'
         config_type = "AWS::ApiGateway::RestApi"
 
+    @property
+    def generate_arn(self):
```

Comment:
generate Arn shouldn't be overwritten anymore, this looks like it was copy pasted, its a bit unclear why.